Scenarios/BlueGreen-Deployment-for-AKS/GAP/bluegreen-aks-solution-content.md

```diff
@@ -0,0 +1,222 @@
+## Overview 
+What happen when you need to upgrade the kubernetes version in a cluster or even better when you need to change kubernetes platform components like: ingress gateway, service mesh, operators and so on, but you don't want to have impact on the workloads/applications that are running on the cluster itself?
+
+The answer that will make everyone happy from business into infra to apps is blue green deployment at Kubernetes infra level.
+With modern principles and availability of cloud services like:
+- [IaC](https://docs.microsoft.com/en-us/devops/deliver/what-is-infrastructure-as-code)
+- [Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure)
+- [Cloud Elasticity](https://azure.microsoft.com/en-us/overview/what-is-elastic-computing/)
+- [Continuous Delivery](https://docs.microsoft.com/en-us/devops/deliver/what-is-continuous-delivery)
+
+Blue Green deployment has become a de-facto standard pattern for the release management and operation at infra and application level in kubernetes environments.
+
+In this article is described the design and implementation of the blue green deployment for AKS leveraging Azure Cloud managed services and native kubernetes features. Adoption of this pattern improves the operation and availability during the deployment of changes/upgrades of AKS clusters.
```

Comment:
Consider changing "In this article is described" to "This articles describes"

Reply:
fixed